api/foxpass_user_cleanup.py

```diff
@@ -50,7 +50,10 @@ def make_list(data):
 def deactivate_user(api_key, user, api_url):
     header = {'Authorization': 'Token ' + api_key}
     r = requests.put(api_url + ENDPOINT + user + '/', headers=header, data=json.dumps(DATA))
-    print('Deactivated {}'.format(user))
+    if r.status_code == 200:
+        print('Deactivated {}'.format(user))
+    else:
+        print('Error deactivating user {}'.format(user))
```

Comment:
Should inform the user about the error which will be sent as part of the API response.